tox.ini

```diff
@@ -18,7 +18,7 @@ pass_env =
     HTTPS_PROXY
 
 [testenv:requirements]
-base_python = python3.8
+base_python = python3.10
```

Comment:
If we change this here (and I don't oppose it), I'd suggest to make `python3.10` the default runner for all the other steps as well.